ui/user/src/lib/components/chat/ChatSidebarConfig.svelte

```diff
@@ -33,7 +32,13 @@
 		{#key layout.mcpServer.id}
 			<div class="flex w-full justify-center px-4 py-4 md:px-8">
 				<div class="flex w-full flex-col gap-4 md:max-w-[1200px]">
-					<ProjectMcpServerTools {project} mcpServer={layout.mcpServer}>
+					<ProjectMcpServerTools
+						{project}
+						mcpServer={layout.mcpServer}
+						submitText="Update"
+						onSubmit={() => closeSidebarConfig(layout)}
```

Comment:
This addresses an issue where hitting the Continue/Update button doesn't close the dialog.